### DISCUSSION TITLE
sed -i -e 's/E_ERROR/E_ALL/g' ~/BirdNET-Pi/scripts/overview.php

### DISCUSSION THREAD

**Discussion by itnetwork** [mcguirepr89/BirdNET-Pi#1205] [Q&A]
In a bit of a pickle. Hoping someone could point me in the right direction.

Long story short. Had an issue where my pi stop detecting birds. It was previous working perfectly for weeks without issue. Tried looking online for similar issues. I came across this post. https://github.com/mcguirepr89/BirdNET-Pi/discussions/731  and ran this command from a putty terminal sed -i -e 's/E_ERROR/E_ALL/g' ~/BirdNET-Pi/scripts/overview.php 

Yes I know I shouldn't run commands I don't understand, bla bla bla. Turns out it was an issue with my new camera rtsp feed. There were many thousand of recordings waiting to be processed. It was detecting them put each files was taking hours to go through. I jumped in and wiped out all the zero kb files and the ones that were malformed. Boom instant life. However, after running the command I'm now getting the following lines on the front page gui

Warning: ini_set(): A session is active. You cannot change the session module's ini settings at this time in /home/admin/BirdNET-Pi/scripts/overview.php on line 4

Warning: session_set_cookie_params(): Cannot change session cookie parameters when session is active in /home/admin/BirdNET-Pi/scripts/overview.php on line 6

Notice: session_start(): A session had already been started - ignoring in /home/admin/BirdNET-Pi/scripts/overview.php on line 7
![Screenshot 2024-05-21 171711](https://github.com/mcguirepr89/BirdNET-Pi/assets/7102215/06591a73-9bf5-4538-a7d3-9bc143b55ce0)

Looking to undo that command if possible and go back to a pretty unblemished website. 

Anyone? Please and thank you in advance.

**Comment by alexbelgium** [comment]
Hi, you could re-download the original file using 
```
rm /home/admin/BirdNET-Pi/scripts/overview.php
curl -o /home/admin/BirdNET-Pi/scripts/overview.php https://raw.githubusercontent.com/mcguirepr89/BirdNET-Pi/main/scripts/overview.php
chown "$USER:$USER" /home/admin/BirdNET-Pi/scripts/overview.php
chmod 777 /home/admin/BirdNET-Pi/scripts/overview.php
```

The chmod command is to set universal permissions as i'm not sure what is needed :) probably a 644 is enough

**Comment by itnetwork** [comment]
Hi Alex,
I really appreciate the information. I can't believe I didn't think of that. It worked perfectly. Again, thank you!
